src/server/models/frame.js

```diff
@@ -72,8 +72,8 @@ class Frame {
         return this.score;
     }
 
-    setScore(score) {
-        if (score < 0 || score > 30 || score == null || typeof score != 'number')
+    setScore(score, nbKeel=10) {
+        if (score < 0 || score > 3*nbKeel || score == null || typeof score != 'number')
```

Comment:
Add a variables at the start of the function for `maxScore = 3 * nbKeel`

Reply:
start of the function or maybe higher as it could be a parameter in the scope of the whole file

Reply:
done